src/process_utility.c

```diff
@@ -47,6 +53,34 @@ prev_ProcessUtility(Node *parsetree,
 	}
 }
 
+static void
+verify_not_chunk(Oid relid, bool allow_overide)
```

Comment:
I think this function name is confusing. Something more like `check_chunk_operation_allowed` maybe ?

Reply:
fixed